cmd/minikube/cmd/config/config.go

```diff
@@ -36,6 +36,7 @@ type setFn func(string, string) error
 type Setting struct {
 	name        string
 	set         func(config.MinikubeConfig, string, string) error
+	setArray    func(config.MinikubeConfig, string, []string) error
```

Comment:
Suggestion for another PR maybe.  The set function should go away and all keys should just be set with `map[key] = val` since its a `map[string]interface{}`.  Not sure why it was done this way in the first place (full disclaimer: I did it).